tests/stub/routing.py

```diff
@@ -664,7 +759,7 @@ def write_tx_script_multiple_bookmarks(self):
         !: AUTO GOODBYE
         !: AUTO RESET
 
-        C: BEGIN {"db": "adb", "bookmarks": ["neo4j:bookmark:v1:tx5", "neo4j:bookmark:v1:tx29", "neo4j:bookmark:v1:tx94", "neo4j:bookmark:v1:tx56", "neo4j:bookmark:v1:tx16", "neo4j:bookmark:v1:tx68"]}
+        C: BEGIN {"db": "adb", "bookmarks{}": ["neo4j:bookmark:v1:tx5", "neo4j:bookmark:v1:tx29", "neo4j:bookmark:v1:tx94", "neo4j:bookmark:v1:tx56", "neo4j:bookmark:v1:tx16", "neo4j:bookmark:v1:tx68"]}
```

Comment:
is it intentional ?

Reply:
Yes. It allows the bookmarks to be sent in arbitrary order.